releasenotes/notes/add-NoiseModel-from_backendproperties-1a3d6d976133a661.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add a new function :func:`from_backend_properties` to :class:`NoiseModel`.
+    This function returns a noise model derived from a device backend properties.
```

Comment:
Again :meth:`~.NoiseModel.from_backend`

It might be good to mention the background why this is added. E.g.
The function allows to construct a noise model from a ``BackendProperties`` object, which can seen as a spin-off of a removed functionality of ``NoiseModel.from_backend``.